cmd/descheduler/app/options/options.go

```diff
@@ -85,7 +85,10 @@ func newDefaultComponentConfig() (*componentconfig.DeschedulerConfiguration, err
 func (rs *DeschedulerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&rs.Logging.Format, "logging-format", "text", `Sets the log format. Permitted formats: "text", "json". Non-default formats don't honor these flags: --add-dir-header, --alsologtostderr, --log-backtrace-at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --skip-headers, --skip-log-headers, --stderrthreshold, --log-flush-frequency.\nNon-default choices are currently alpha and subject to change without warning.`)
 	fs.DurationVar(&rs.DeschedulingInterval, "descheduling-interval", rs.DeschedulingInterval, "Time interval between two consecutive descheduler executions. Setting this value instructs the descheduler to run in a continuous loop at the interval specified.")
-	fs.StringVar(&rs.KubeconfigFile, "kubeconfig", rs.KubeconfigFile, "File with  kube configuration.")
+	fs.StringVar(&rs.ClientConnection.Kubeconfig, "kubeconfig", rs.ClientConnection.Kubeconfig, "File with kube configuration. Deprecated, use client-connection-kubeconfig instead.")
+	fs.StringVar(&rs.ClientConnection.Kubeconfig, "client-connection-kubeconfig", rs.ClientConnection.Kubeconfig, "File path to kube configuration for interacting with kubernetes apiserver.")
+	fs.Float32Var(&rs.ClientConnection.QPS, "client-connection-qps", rs.ClientConnection.QPS, "QPS to use for interacting with kubernetes apiserver.")
+	fs.Int32Var(&rs.ClientConnection.Burst, "client-connection-burst", rs.ClientConnection.Burst, "Burst to use for interacting with kubernetes apiserver.")
```

Comment:
do we need policy fields and config flags for this?

Reply:
Can you please elaborate on policy fields? I was under the assumption that defining it at `DeschedulerConfiguration` is sufficient.

Reply:
I mean the new ClientConnection field in `DeschedulerPolicy`, along with these flags, gives 2 ways to configure these settings. Is that right? My question is, should this instead just be one or the other

Reply:
I see. I was under the impression that we provide both when I looked at things such as leader-election. Happy to remove the cli flags, if you suggest.

Reply:
Ah I assumed by your updates to the [readme](https://github.com/kubernetes-sigs/descheduler/pull/1034/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R841) that you had added these fields to [`DeschedulerPolicy`](https://github.com/kubernetes-sigs/descheduler/blob/e6580b97a854594d2fd0c7d8f31d53ceb1bde7f3/pkg/api/v1alpha1/types.go#L26) too.

This is fine as it is, you just need to change the readme update because this can't be set through the DeschedulerPolicy config yaml